src/main/java/Brad/Brad.java

```diff
@@ -0,0 +1,253 @@
+package Brad;
+
+import Brad.Exceptions.emptyArgumentException;
+import Brad.Exceptions.invalidNumberException;
+import Brad.Exceptions.dataCorruptedException;
+import Brad.Tasks.List;
+import Brad.Tasks.TaskType;
+import java.io.FileNotFoundException;
+import java.io.IOException;
+
+import java.util.Scanner;
+
+
+public class Brad {
+    private static List inputList = new List();
```

Comment:
Try to use a better descriptive name, maybe like tasklist (as this is a list of tasks)